cmd/antrea-agent/options.go

```diff
@@ -185,7 +192,10 @@ func (o *Options) validateFlowExporterConfig() error {
 			return fmt.Errorf("IPFIX flow collector address should be provided")
 		} else {
 			// Check if it is TCP or UDP
-			strSlice := strings.Split(o.config.FlowCollectorAddr, ":")
+			strSlice, err := parseFlowCollectorAddr(o.config.FlowCollectorAddr)
```

Comment:
Maybe add a comment in antrea-agent.conf about IPv6 is supported for flowCollectorAddr.

Reply:
Updated.